packages/react/src/experimental/Banners/F0SdmCallout/SdmCalloutInternal.tsx

```
@@ -0,0 +1,166 @@
+import { Button } from "@/components/Actions/Button"
+import { F0Icon, IconType } from "@/components/F0Icon"
+import { OneEllipsis } from "@/components/OneEllipsis"
+import { RichTextDisplay } from "@/experimental/RichText/RichTextDisplay"
+import { CheckDouble, Cross, InfoCircle, Warning } from "@/icons/app"
+import { cn } from "@/lib/utils"
+import { Skeleton } from "@/ui/skeleton"
+import { cva } from "cva"
+import { forwardRef } from "react"
+import { SdmCalloutInternalProps, SdmCalloutSkeletonProps } from "./types"
+
+const sdmCalloutVariants = cva({
+  base: "flex w-full flex-col rounded-lg p-[1px]",
+  variants: {
+    variant: {
+      default:
+        "bg-gradient-to-l from-[#A1ADE51F] via-[#E519431F] to-[#E556191F]",
+      critical: "bg-f1-background-critical",
+      positive: "bg-f1-background-positive",
+      info: "bg-f1-background-info",
+      warning: "bg-f1-background-warning",
+    },
+  },
+  defaultVariants: {
+    variant: "default",
+  },
+})
+
+const variantIcons: Record<string, IconType> = {
+  positive: CheckDouble, // Using CheckDouble as the primary action icon for positive
+  warning: Warning,
+  info: InfoCircle,
+}
+
+const variantTitleColors: Record<string, string> = {
+  positive: "text-f1-foreground-positive font-medium",
+  warning: "text-f1-foreground-warning font-medium",
+  info: "text-f1-foreground-info font-medium",
+}
+
+export const SdmCalloutInternal = forwardRef<
+  HTMLDivElement,
+  SdmCalloutInternalProps
+>(function SdmCalloutInternal(
+  { title, onClose, content, calloutActions = [], variant = "default" },
+  ref
+) {
+  // Limit actions to maximum 2
+  const limitedActions = calloutActions.slice(0, 2)
```

Instead of slicing the array I would just throw an error here in the case of this array containing more than 2 elements